health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -279,14 +371,14 @@ const ViewHierarchy = () => {
                                                 <div style={{fontWeight:"600", fontSize:"1.2rem"}}>
                                                     {/* {t(hierItem?.boundaryType)} */}
-                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toUpperCase())}`}
+                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toLowerCase().replace(/\s+/g, "_"))}`}
                                                 </div>
                                                 <div style={{height:"1rem"}}></div>
                                                 <Card type={"primary"} variant={"form"} className={"question-card-container"} >
                                                     <div style={{display:"flex", gap:"2rem"}}>
                                                     <Svgicon />
                                                     <div style={{display:"flex", alignItems:"center", fontWeight:"600"}}>
-                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toUpperCase())}`}{"-geojson.json"}
+                                                    {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toLowerCase().replace(/\s+/g, "_"))}`}{"-geojson.json"}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)